### PR TITLE
Updating Cloudinary Video Resources

### PR DESCRIPTION
**Pull request by colbyfayock**
### What?

Updates the Cloudinary bullets and links under the Video Optimization.

https://nextjs.org/docs/app/building-your-application/optimizing/videos#cloudinary-integration

### Why?

More relevant direct link to the Cloudinary-supported Next.js SDK along with an additional link to the Video solution itself.

### CODE REVIEW

File: docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx [hunk 1]
```diff
@@ -257,8 +257,10 @@ Explore these video streaming platforms for integrating video into your Next.js
 
 ### Cloudinary Integration
 
-- Official [documentation and integration guides](https://support.cloudinary.com/hc/en-us/categories/200477762-Client-Libraries-and-Integration-Guides) for using Cloudinary with Next.js.
-- A [specific guide](https://next.cloudinary.dev/nextjs-13) on integrating Cloudinary with Next.js.
+- [Documentation and integration guide](https://next.cloudinary.dev/) for using Cloudinary with Next.js (Next Cloudinary)
+- Includes a `<CldVideoPlayer>` component for [drop-in video support](https://next.cloudinary.dev/cldvideoplayer/basic-usage)
```
Comment (samcx) [inline docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx:261]:
```suggestion
- Includes a `<CldVideoPlayer>` component for [drop-in video support](https://next.cloudinary.dev/cldvideoplayer/basic-usage).
```

File: docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx [hunk 2]
```diff
@@ -257,8 +257,10 @@ Explore these video streaming platforms for integrating video into your Next.js
...
+- Find [examples](https://github.com/cloudinary-community/cloudinary-examples/?tab=readme-ov-file#nextjs) of integrating Cloudinary with Next.js including [Adaptive Bitrate Streaming](https://github.com/cloudinary-community/cloudinary-examples/tree/main/examples/nextjs-cldvideoplayer-abr)
```
Comment (samcx) [inline docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx:262]:
```suggestion
- Find [examples](https://github.com/cloudinary-community/cloudinary-examples/?tab=readme-ov-file#nextjs) of integrating Cloudinary with Next.js including [Adaptive Bitrate Streaming](https://github.com/cloudinary-community/cloudinary-examples/tree/main/examples/nextjs-cldvideoplayer-abr).
```

**Review by samcx — CHANGES_REQUESTED** [review]

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 10e82aa01d39eaa158911269c4f39913b91d800f

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 10e82aa01d39eaa158911269c4f39913b91d800f

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by samcx — APPROVED** [review]
Thank you for submitting a <picture data-single-emoji=":pr:" title=":pr:">:pr:</picture>!

<picture data-single-emoji=":lgtm:" title=":lgtm:">:lgtm:</picture>

**Comment by colbyfayock** [comment]
hey just checking in to make sure you're not waiting on me for anything! thanks

**Comment by samcx** [comment]
@colbyfayock Not waiting for anything! Looks like this fell in the test blackhole—will re-run this and hopefully it merges!

**Comment by colbyfayock** [comment]
haha thanks @samcx 🙏

**Pull request merged**